troubleshooting/known_issues/check_system_management_monitoring_tools.md

```diff
@@ -0,0 +1,17 @@
+# 1.3 Check of system management monitoring tools
+
+If all designated prerequisites are met, the availability of system management health services may optionally be validated by accessing the URLs listed in
+[Access System Management Health Services](../../operations/system_management_health/Access_System_Management_Health_Services.md).
+It is very important to check the `Prerequisites` section of this document.
```

Comment:
Prerequisites section of which document? There isn't one in this document that I can see. If it's in the linked document above, then I recommend linking directly to that section here, to prevent confusion about which document is being referenced.